apps/site/types/partners.ts

```diff
@@ -1,6 +1,6 @@
 import type { ReactElement, SVGProps } from 'react';
 
-export type Partners = {
+export type Partner = {
   id: string;
```

Comment:
We could just set the id to `keyof typeof PartnerLogos` instead of asserting it each time we want to use it as a key. We have the values hardcoded anyway, so I don't think it would cause any issues + would provide some nice type completion

Reply:
I can’t quite understand what you mean, could you show it as a code suggestion, please?

Reply:
```suggestion
  id: keyof typeof PartnerLogos;
```

Then we could remove the assertions from `apps/site/components/Common/Partners/index.tsx` lines 47 and 62